llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/maps/base.py

```diff
@@ -92,17 +95,16 @@ def load_data(
                     ]
                 )
                 place_text = f"Place: {place.display_name}, Address: {place.address}, Average Rating: {place.average_rating}, Number of Ratings: {place.number_of_ratings}"
-                text = f"{place_text}\n{reviews_text}"
+                document_text = f"{place_text}\n{reviews_text}"
 
                 if len(documents) == number_of_results:
                     return documents
 
-                if len(text) > MAX_TEXT_LENGTH:
-                    text = text[:MAX_TEXT_LENGTH]
-                documents.append(Document(text=text, extra_info=place.dict()))
-
+                if len(document_text) > MAX_TEXT_LENGTH:
+                    document_text = document_text[:MAX_TEXT_LENGTH]
```

Comment:
(probably these constants should be user-configurable, but that could be another PR)